mailpoet/lib/Migrations/Migration_20230712_180341.php

```diff
@@ -0,0 +1,53 @@
+<?php declare(strict_types = 1);
+
+namespace MailPoet\Migrations;
+
+use MailPoet\Entities\DynamicSegmentFilterData;
+use MailPoet\Migrator\Migration;
+use MailPoet\Segments\DynamicSegments\DynamicSegmentFilterRepository;
+use MailPoet\Segments\DynamicSegments\Filters\WooCommerceAverageSpent;
+use MailPoet\Segments\DynamicSegments\Filters\WooCommerceNumberOfOrders;
+use MailPoet\Segments\DynamicSegments\Filters\WooCommerceSingleOrderValue;
+use MailPoet\Segments\DynamicSegments\Filters\WooCommerceTotalSpent;
+
+class Migration_20230712_180341 extends Migration {
+  public function run(): void {
+    $dynamicSegmentFilterRepository = $this->container->get(DynamicSegmentFilterRepository::class);
+    $filters = $dynamicSegmentFilterRepository->findBy(
+      [
+        'filterData.action' => [
+          WooCommerceNumberOfOrders::ACTION_NUMBER_OF_ORDERS,
+          WooCommerceTotalSpent::ACTION_TOTAL_SPENT,
+          WooCommerceSingleOrderValue::ACTION_SINGLE_ORDER_VALUE,
+          WooCommerceAverageSpent::ACTION,
+        ],
+      ]
+    );
+
+    foreach ($filters as $filter) {
+      $filterData = $filter->getFilterData();
+      $data = $filter->getFilterData()->getData();
+
+      if (isset($data['number_of_orders_days'])) {
+        $days = $data['number_of_orders_days'];
+      } else if (isset($data['total_spent_days'])) {
```

Comment:
Just curious, is there a reason you're preferring `else if` to `elseif` here? It looks like the [WP standard](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#use-elseif-not-else-if) is to use `elseif`, but of course we aren't following WP standards in general.

Reply:
No reason other than muscle memory. Despite the WP coding standard, I'm more used to `else if` than `elseif`. Happy to change it if you prefer. I'm not sure if it is worth it or not.

Reply:
No preference here, just curious!